src/Sandbox.js

```diff
@@ -20,16 +19,18 @@ const Sandbox = () => {
         <fieldset>
           <legend>Try out an endpoint:</legend>
           <div id="user-input">
-            <label htmlFor="endpoint-input">/api/v1/characters</label>
+            <label htmlFor="endpoint-input">
+              Finish the URL /api/v1/characters/
```

Comment:
I added the `/` back at the end because I found that the fetch is automagically adding it so if you type `/` in the input then there's an error because `//`

Reply:
Ooh interesting! Good find!